### PR TITLE
Add visible create / updated dates

### PR DESCRIPTION
**Pull request by andreban**
- Add created and updated fields on Pwa constructor.
- Update the updated field when a Pwa is updated.

### CODE REVIEW

File: views/pwas/view.hbs [hunk 1]
```diff
@@ -48,6 +48,8 @@
         <div class="pwabox-body">
           <h3>{{pwa.name}}</h4>
           <p>{{pwa.description}}</p>
+          <p>Added: {{pwa.created}}</p>
```
Comment (andreban) [inline views/pwas/view.hbs]: How should we format the dates? Is just year, month and day enough or we also want to add hour + minute?

Comment (ithinkihaveacat) [inline views/pwas/view.hbs]: moment.js [does relative dates](http://momentjs.com/docs/#/displaying/fromnow/), if you want to escalate to that…

Comment (andreban) [inline views/pwas/view.hbs]: That looks like a good one. "Created 5 days ago" definitely reads better.

**Comment by andreban** [comment]
Added `moments.js` to format the dates. Looks awesome now. Thanks @ithinkihaveacat

**Comment by ithinkihaveacat** [comment]
LGTM

**Pull request merged**